### ISSUE TITLE
 Executive Summary & Deliverables

### ISSUE THREAD

**Issue by multiminparvo** [bounswe/bounswe2019group5#43]
Writing Executive Summary, List and status of deliverables

Link: https://docs.google.com/document/d/1bHIoHnf053GHFogreFLzmKIvIytHYwiec7G3zpwUg3Y/edit?usp=sharing

**Comment by Melteos** [comment]
I think you can add
 1.Repository Research
2.Communication Plan
(and maybe 3.)Label Philosophy https://github.com/bounswe/bounswe2019group5/wiki/Label-Philosophy
to the table of deliverables because in the pdf "1. Project repository with project documents and issue management" is also a deliverable.

**Comment by multiminparvo** [comment]
I saw your comment really late, sorry for that.
We discussed with @ymBila  about the the topics that you mentioned above.

1-) Since repository search is not directly related with our project, we didn't want to add it.
2-) There should have been already communication plan at the part of the our report so we excluded it
3-) We examined last year reports, they were written not very detailed way about the label thing etc. 
If we add the label philosophy part, our reports will be too messy. we want to keep it simple as much as possible  

Thanks for your feedback, we can discuss these issues on our meeting :)
